<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/1.png);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="cadenaEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetAutomata" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>531</width>
      <height>511</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="nuevoEstado">
       <property name="text">
        <string>NUEVO ESTADO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nuevoFinal">
       <property name="text">
        <string>NUEVO EST. FINAL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="eliminarTransicion">
       <property name="text">
        <string>ELIMINAR TRANSICION</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="botonZoomMas">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonZoomMenos">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonTransicion">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>57</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>MEDIA/arrow.png</normaloff>MEDIA/arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>10</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="botonEstado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>57</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>MEDIA/nodo.png</normaloff>MEDIA/nodo.png</iconset>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>351</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="TuplaAutomata">
     <attribute name="title">
      <string>TUPLA AUTOMATA</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>341</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>ESTADOS FINALES:               ELEMENTOS: </string>
      </property>
      <widget class="QTableWidget" name="tablaEstadosFinales">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>151</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QTableWidget" name="tablaEstadosFinales_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>181</width>
         <height>151</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>341</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>TRANSICIONES</string>
      </property>
      <widget class="QTableWidget" name="TablaTransiciones">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>321</width>
         <height>321</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>ESTADO Ini.</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ELEMENTO</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ESTADO Fin.</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>DETALLE ESTADO</string>
     </attribute>
     <widget class="QLabel" name="labelEstado">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>ESTADO ID : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>331</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Transiciones Salientes</string>
      </property>
      <widget class="QTableWidget" name="tablaSaliente">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>311</width>
         <height>192</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>ELEMENTO</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ESTADO</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>331</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Transiciones Entrantes</string>
      </property>
      <widget class="QTableWidget" name="tablaEntrante">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>311</width>
         <height>211</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>ESTADO</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ELEMENTO</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="checkLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>MEDIA/noCheck.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>botonEstado</sender>
   <signal>clicked()</signal>
   <receiver>eliminarTransicion</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonTransicion</sender>
   <signal>clicked()</signal>
   <receiver>nuevoFinal</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonTransicion</sender>
   <signal>clicked()</signal>
   <receiver>nuevoEstado</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonEstado</sender>
   <signal>clicked()</signal>
   <receiver>nuevoEstado</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonEstado</sender>
   <signal>clicked()</signal>
   <receiver>nuevoFinal</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonTransicion</sender>
   <signal>clicked()</signal>
   <receiver>eliminarTransicion</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
